WcaOnRails/lib/relations.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+LINKINGS = ActiveRecord::Base.connection.execute("SELECT wca_id, wca_ids FROM linkings")
```

Comment:
We discussed this offline, but just putting it here for the world to see: this is too permanent. We shouldn't have to restart the server just to pull in updates here. I assume you'll fix this when you add in a background job to periodically compute the auxiliary table you created.